pkg/operator/reporting/validate.go

```diff
@@ -300,33 +300,6 @@ func GetDependentGenerationQueriesWithDataSourcesMemoized(queryGetter reportGene
 	return nil
 }
 
-func GetDependentGenerationQueriesMemoized(queryGetter reportGenerationQueryGetter, generationQuery *metering.ReportGenerationQuery, depth, maxDepth int, queriesAccumulator map[string]*metering.ReportGenerationQuery, dynamicQueries bool) error {
```

Comment:
to verify, this is just dead code right? looks so from grepping the repo.

Reply:
Yeah